provider/marathon/builder_test.go

```diff
@@ -0,0 +1,131 @@
+package marathon
+
+import "github.com/gambol99/go-marathon"
+
+// Functions related to building applications.
+
+func createApplication(ops ...func(*marathon.Application)) marathon.Application {
```

Comment:
The conventional way in Java it's a `build` prefix.

- https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.Builder.html
- https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.Builder.html
- https://github.com/spring-projects/spring-framework/blob/bc14c5ba83e1f211628456bbccce7b2531aac58c/spring-test/src/test/java/org/springframework/test/context/support/BootstrapTestUtilsMergedConfigTests.java
- https://github.com/spring-projects/spring-framework/blob/7018804e8451fa935e8715754de43a3ef62fa5c8/spring-webflux/src/test/java/org/springframework/web/reactive/function/server/RequestPredicatesTests.java

I prefer you simply remove the `create` prefix.